meteor/sofie-server/model/entities/rundown.ts

```diff
@@ -139,9 +139,10 @@ export class Rundown extends BasicRundown {
 	}
 
 	public deactivate(): void {
-		this.assertActive(this.deactivate.name)
-		this.activeSegment.takeOffAir()
-		this.activePart.takeOffAir()
+		this.assertActive('deactivate')
+		//TODO: Remove the if-statements once we save the activeSegment and activePart to the database
```

Comment:
So we are able to remove the if-statements now?

Reply:
A great point :-D They will be removed, testing seems to prove no errors doing that.